duckdns/DOCS.md

```diff
@@ -115,6 +115,8 @@ aliases:
     alias: my-domain.duckdns.org
 ```
 
+> Note that the `domain` key should receive **your custom domain name** and the `alias` key should rececive the DuckDNS domain name
```

Comment:
```suggestion
Notice that here `domain` is set to a a custom domain name, not a duckdns one. The duckdns domain this custom domain is an alias for goes in `alias`.
```
This isn't quoting anything so it shouldn't use that formatting. There are other examples of `Note: ` formatting in this doc but I actually don't think this needs it. I think we can just rephrase it as part of the example's explanation like so.